doc/changes/changes_0.1.0.md

```diff
@@ -7,7 +7,7 @@ In this first release we created an docker test environment for Exasol. The test
 ### Currently supported Operating Systems
 
 * Linux
-* Mac OS X with Intel processors (please consult the [README](../../README.md) for more details)
+* Mac OS X with Intel processors (please consult the [User Guide](../user_guide/user_guide.rst) for more details)
```

Comment:
hm, this will be a bit ugly with sphinx, we need to check how you can do sphinx refs with myst_parser

Reply:
Apparently this section was removed in all later changelogs. I believe it's more correct to revert this change, as in version 0.1.0 the info about MacOsX was described in README.md.

Reply:
ah, I see, yes please revert